docs/cadc/cadc.rst

```diff
@@ -0,0 +1,564 @@
+  >>> result = cadc.query_region('08h45m07.5s +54d18m00s')
+  >>> print(len(result))
+
+        662
+
+  >>> urls = cadc.get_data_urls(result[result['target_name'] == 'Nr3491_1'],
+                                include_auxiliaries=True)
+  >>> for url in urls:
+  >>>    print(url)
+
+    https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/data/pub/CFHT/2376828o_preview_zoom_1024.jpg?RUNID=bxpv43misqekd16f
+    https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/data/pub/CFHT/2376828o.fits.fz?RUNID=bxpv43misqekd16f
+    https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/data/pub/CFHT/2376828o_preview_1024.jpg?RUNID=bxpv43misqekd16f
+    https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/data/pub/CFHT/2376828o_preview_256.jpg?RUNID=bxpv43misqekd16f
+
+
+CADC data can also be queried on the target name. Note that the name is not
+resolved. Instead it is matched against the target name in the CADC metadata.
+
+.. code-block:: python
+
+  >>> from astroquery.cadc import Cadc
+  >>>
+  >>> cadc = Cadc()
+  >>> result = cadc.query_name('M31)
+  >>> print(len(result))
+
+    2000
+
+  >>> result = cadc.query_name('Nr3491_1')
+  >>> print(result)
+
+    caomObservationURI sequenceNumber proposal_keywords target_standard ... energy_transition_transition          time_bounds [2]          polarization_states      lastModified_2
+                                                                        ...                                              d
+    ------------------ -------------- ----------------- --------------- ... ---------------------------- --------------------------------- ------------------- -----------------------
+     caom:CFHT/2376828        2376828                                 0 ...                              58546.328009 .. 58546.32960815509                     2019-03-04T08:14:46.470
+
+
+Note that the examples above are for accessing data anonymously. Users with
+access to proprietary data can call ```login``` on the ```cadc``` object
+before querying or accessing the data.
+
+CADC metadata is available through a TAP service. While the above interfaces
+offer a quick and simple access to the data, the TAP interface presented in
+the next sections allows for more complex queries.
+
+=============================
+Query CADC metadata using TAP
+=============================
+
+Cadc TAP access is based on a TAP+ REST service. TAP+ is an extension of
+Table Access Protocol (TAP: http://www.ivoa.net/documents/TAP/) specified by the
+International Virtual Observatory Alliance (IVOA: http://www.ivoa.net).
+
+The TAP query language is Astronomical Data Query Language
+(ADQL: http://www.ivoa.net/documents/ADQL/2.0), which is similar
+to Structured Query Language (SQL), widely used to query databases.
+
+TAP provides two operation modes: Synchronous and Asynchronous:
+
+* Synchronous: the response to the request will be generated as soon as the
+  request received by the server.
+  (Do not use this method for queries that generate a big amount of results.)
```

Comment:
it would be helpful to guide users as to what qualifies as 'big'

Reply:
Re-worked the phrase